src/JSPool.Tests/JsPoolTests.cs

```diff
@@ -21,7 +21,7 @@ public class JsPoolTests
 		public void ConstructorCreatesEngines()
 		{
 			var factory = new Mock<IEngineFactoryForMock>();
-			factory.Setup(x => x.EngineFactory()).Returns(new Mock<IJsEngine>().Object);
+			factory.Setup(x => x.EngineFactory()).Returns(() => new Mock<IJsEngine>().Object);
```

Comment:
Were these tests broken before? I haven't seen any failures, but maybe the tests weren't running :/